docs/src/manifolds/power.md

```diff
@@ -1,5 +1,56 @@
 # Power manifold
 
+A power manifold is based on a [`Manifold`](@ref) $\mathcal M$ to build a
+$\mathcal M^{n_1 \times n_2 \times \cdots \times n_m}$.
+For the two special cases $m=1$ and $m=2$ we obtain
+one-dimensional data, i.e. vectors of points on $\mathcal M$ and matrices of images,
+respectively.
+
+## Example
+There are two ways to store the data: in a multidimensional array or in a nested array.
+
+Let's look at both for an example. Let $\mathcal M$ be `Sphere(2)` the 2-sphere. and we want
```

Comment:
```suggestion
Let's look at both for an example. Let $\mathcal M$ be `Sphere(2)` the 2-sphere and we want
```

Reply:
I rephrased also the first one when adding your suggestion.